Config/Sample.cfg

```diff
@@ -105,6 +105,8 @@ FPSInVPS                        : 0             # 0: OFF, 1: Enable VPS timing i
 MaxCLL                          : 0             # SEI message (0=OFF,  1=ON ) 
 MaxFALL                         : 0             # SEI message (0=OFF,  1=ON )
 MasterDisplay                   : NULL          # SEI message. Signals SMPTE ST 2086 display info in the format "G(%hu,%hu)B(%hu,%hu)R(%hu,%hu)WP(%hu,%hu)L(%u,%u)" where %hu,%u are unsigned 16bit and 32 bit integers
+DolbyVisionProfile              : 0             # SEI message (0=OFF, 8.1 or 81 : Send bitstreams compliant to Dolby Vision Profile 81)
+DolbyVisionRpuFile              : NULL          # SEI message. Dolby Vision RPU metadata file path.
```

Comment:
file should have a file name not null

Reply:
Please correct me if I am wrong, actually we are adding the default values of the parameters in the Sample.cfg right?

Reply:
Correct: it's just that NULL is already considered a string and the application will be reading it as a non NULL string. so it's best to have another value (such as an example) or a sample file to read from.